packages/editor/src/components/blocks/datatablecolumn/DataTableColumn.tsx

```diff
@@ -0,0 +1,45 @@
+import type { DataTableColumn, Prettify } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, UiComponentProps } from '../../../types/config';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@axonivy/ui-components';
+
+type DataTableColumnProps = Prettify<DataTableColumn>;
+
+export const defaultDataTableColumnProps: DataTableColumn = {
+  header: 'header',
+  value: 'value',
+  ...defaultBaseComponent
+} as const;
+
+export const DataTableColumnComponent: ComponentConfig<DataTableColumnProps> = {
+  name: 'DataTableColumn',
+  category: 'Elements',
+  subcategory: 'Input',
+  icon: '',
+  description: 'A Column for the DataTable',
+  defaultProps: defaultDataTableColumnProps,
+  render: props => <UiBlock {...props} />,
+  create: ({ label, value }) => ({ ...defaultDataTableColumnProps, header: label, value }),
+  outlineInfo: component => component.header,
+  fields: {
+    header: { subsection: 'General', label: 'Header', type: 'text' },
+    value: { subsection: 'General', label: 'Value', type: 'textBrowser', options: { onlyAttributes: 'COLUMN' } },
+    ...baseComponentFields
+  },
+  hidden: true
```

Comment:
Better, but still not 100% was I had in mind.
I meant that you instead of add this block to the category 'Elements', add it to a category 'Hidden'.